cmd/cli/plugin/feature/README.md

```diff
@@ -45,26 +45,28 @@ Use "tanzu feature [command] --help" for more information about a command.
 
 ```sh
 >>> tanzu feature list --help
-List Features
+List features
 
 Usage:
   tanzu feature list [flags]
 
 Examples:
   
-    # List a clusters Features
+    # List the features referenced by (a) featuregate(s).
     tanzu feature list --activated
     tanzu feature list --unavailable
     tanzu feature list --deactivated
 
 Flags:
-  -a, --activated            List only activated Features
-  -d, --deactivated          List only deactivated Features
-  -e, --extended             Include extended output. Higher latency as it requires more API calls.
-  -f, --featuregate string   List Features gated by a particular FeatureGate (default "tkg-system")
-  -h, --help                 help for list
-  -o, --output string        Output format (yaml|json|table)
-  -u, --unavailable          List only Features specified in the gate but missing from cluster
+  -a, --activated                             List only activated features
+  -d, --deactivated                           List only deactivated features
+  -e, --extended                              Include extended output
+  -x, --include-experimental                  Allows displaying experimental features
+  -f, --featuregate string                    List features gated by a particular FeatureGate
+      --permanentlyVoidAllSupportGuarantees   Allow activating a non-stable Feature that will permanently void warranty support. Use with caution.
```

Comment:
Why do we need this option for listing features? Or this is a documentation error?

Reply:
`--permanentlyVoidAllSupportGuarantees` is required per Feature Graduation Process design.

Reply:
Oh, you're right. I shouldn't have this in the `feature list` command documentation. It's only applicable for `feature activate` command.